ecommerce/views.py

```diff
@@ -275,3 +277,12 @@ def post(self, request, code, *args, **kwargs):  # pylint: disable=unused-argume
                     'coupon': CouponSerializer(coupon).data,
                 }
             )
+
+
+class PaymentCallBackView(RedirectView):
+    """
+    payment callback view that will redirect to dashboard
+    """
+    url = DASHBOARD_URL
+    query_string = True
+    permanent = True
```

Comment:
This shouldn't be permanent because the browser could cache it which would probably work without issue now but could cause issues later if that changes.